conda_build/config.py

```diff
@@ -48,6 +48,7 @@ def set_invocation_time():
 ignore_verify_codes_default = []
 exit_on_verify_error_default = False
 conda_pkg_format_default = None
+extra_info_default = 'false'
```

Comment:
Why is this quoted but other boolean values not?

Reply:
I was actually kind of confused by this too, it seems like `exit_on_verify_error_default` is the only one that isn't [quoted](https://github.com/conda/conda-build/pull/4304/files/b4e527a832cb70fccbbb30bf22ad820072321c2c#diff-eaaf464431ee59c1575fecf5aa57498e16bb0694fd2cb6ee1638d3ebd682e68fR41-R51).